tests/test_configuration_cli.py

```diff
@@ -0,0 +1,167 @@
+#
+# Sonar Scanner Python
+# Copyright (C) 2011-2024 SonarSource SA.
+# mailto:info AT sonarsource DOT com
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+#
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+import unittest
+
+from unittest.mock import patch
+from io import StringIO
+
+from pysonar_scanner.configuration import CliConfigurationLoader
+
+
+class TestCliConfigurationLoader(unittest.TestCase):
+    def setUp(self):
+        self.maxDiff = None
+
+    @patch("sys.argv", ["myscript.py"])
+    def test_missing_cli_args(self):
```

Comment:
Related to my other comment, I'm not sure this test makes sense: IMO no parameter should strictly be required from the CLI (they should be require from the `ConfigurationLoader`, though - for which it would be good to have a test ensuring the mandatory parameters are provided, one way or another.